erpnext/accounts/doctype/shipping_rule/shipping_rule.py

```diff
@@ -54,13 +54,14 @@ def sort_shipping_rule_conditions(self):
 			d.idx = i + 1
 
 	def validate_overlapping_shipping_rule_conditions(self):
-		def overlap_exists_between((x1, x2), (y1, y2)):
+		def overlap_exists_between(x, y):
 			"""
 				(x1, x2) and (y1, y2) are two ranges
 				if condition x = 100 to 300
 				then condition y can only be like 50 to 99 or 301 to 400
 				hence, non-overlapping condition = (x1 <= x2 < y1 <= y2) or (y1 <= y2 < x1 <= x2)
 			"""
+			(x1, x2), (y1, y2) = x, y
 			separate = (x1 <= x2 <= y1 <= y2) or (y1 <= y2 <= x1 <= x2)
 			return (not separate)
```

Comment:
I meant in the return statement --> `return not separate` instead of `return(not separate)`. It's just a code clean up similar to removing the extra spaces